src/bindfs.1

```diff
@@ -174,6 +174,14 @@ to distinguish specifier parts, e.g. \fIa:pci:0000:00:00/...\fP,
 where \fIa\fP is an "any type" specifier and \fIpci:0000:00:00\fP is
 a file name.
 
+.TP
+.B \-\-delete\-filtered
+When removing a directory, also transparently remove hidden with
+\fB\-\-file\-filter\fP files in it. If this option not active,
+such directories won't remove with \fBENOEMPTY\fP error. If a directory
+contains hidden directories, they will be deleted recursively, as with
+\fBrm -rf\fP command. If \fB\-\-resolve\-symlinks\fP is active, hidden
+files in resolved directories will not be removed.
```

Comment:
This option is intended to, returning to my QEMU use case, delete directories which contains hidden metadata dirs from resulting share, otherwise rmdir() fails with ENOEMPTY